docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -228,46 +200,80 @@ If you encounter network issues when installing IPEX, you can also install BigDL
 
       .. code-block:: bash
 
-         # get the wheels on Windows system
-         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.1.0a0%2Bcxx11.abi-cp39-cp39-win_amd64.whl
-         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.16.0a0%2Bcxx11.abi-cp39-cp39-win_amd64.whl
-         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.1.10%2Bxpu-cp39-cp39-win_amd64.whl
+         # get the wheels on Linux system for IPEX 2.0.110+xpu
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.0.1a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.15.2a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.0.110%2Bxpu-cp39-cp39-linux_x86_64.whl
 
       Then you may install directly from the wheel archives using following commands:
 
-      .. code-block:: cmd
+      .. code-block:: bash
 
          # install the packages from the wheels
-         pip install torch-2.1.0a0+cxx11.abi-cp39-cp39-win_amd64.whl
-         pip install torchvision-0.16.0a0+cxx11.abi-cp39-cp39-win_amd64.whl
-         pip install intel_extension_for_pytorch-2.1.10+xpu-cp39-cp39-win_amd64.whl
+         pip install torch-2.0.1a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
+         pip install torchvision-0.15.2a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
+         pip install intel_extension_for_pytorch-2.0.110+xpu-cp39-cp39-linux_x86_64.whl
 
          # install bigdl-llm for Intel GPU
          pip install --pre --upgrade bigdl-llm[xpu]
 
-   .. tab:: PyTorch 2.0 Linux
+```
+
+### Runtime Configuration
+
+To use GPU acceleration on Linux, several environment variables are required or recommended before running a GPU example.
+
+```eval_rst
+.. tabs::
+   .. tab:: Intel Arc™ A-Series Graphics and Intel Data Center GPU Flex Series
+
+      For Intel Arc™ A-Series Graphics and Intel Data Center GPU Flex Series, we recommend:
 
       .. code-block:: bash
 
-         # get the wheels on Linux system for IPEX 2.0.110+xpu
-         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.0.1a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
-         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.15.2a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
-         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.0.110%2Bxpu-cp39-cp39-linux_x86_64.whl
+         # configures OneAPI environment variables
+         source /opt/intel/oneapi/setvars.sh
 
-      Then you may install directly from the wheel archives using following commands:
+         export USE_XETLA=OFF
+         export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+
```

Comment:
As we discussed before, I feel it would better to split `Runtime Configuration` into `source OneAPI` & `BKC` two small sections and make user clear that `source /opt/intel/oneapi/setvars.sh` is necessary and other env setting are used for performance improvement.